data-serving/data-service/schemas/cases.schema.json

```diff
@@ -355,6 +355,33 @@
           }
         }
       },
+      "transmission": {
+        "route": {
+          "enum": [
+            "Airborne infection",
+            "Droplet infection",
+            "Fecal–oral",
+            "Sexual",
+            "Oral",
+            "Direct contact",
+            "Vertical",
+            "Iatrogenic",
+            "Vector borne",
+            "Other",
+            "Unknown"
+          ]
+        },
+        "place": {
+          "bsonType": "string"
+        },
+        "linkedCases": {
```

Comment:
Will this be the IDs of the cases? If so, could we clarify this in the field name (linkedCaseIds)?